app/services/data/export/convert-to-csv.service.js

```diff
@@ -8,42 +8,31 @@
 /**
  * Converts data to a CSV formatted string
  *
- * @param {String[]} headers An array containing the column names
- * @param {[[*]]} rows An array of arrays each representing a row from the table
+ * @param {[]} data An array representing either the headers or rows from a db table
  *
  * @returns {String} A CSV formatted string
  */
-function go (headers, rows) {
-  const transformedHeaders = _transformDataToCSV([headers])[0]
-
-  if (!rows) {
-    return transformedHeaders
+function go (data) {
+  if (!data) {
+    return
```

Comment:
Do we want this to return an empty string instead of `undefined`? Because then we're consistent with the type that the service returns?